### ISSUE TITLE
update readme

### ISSUE THREAD

**Issue by MichaelDimmitt** [ckchessmaster/ServerBrowser#1]
The code base says that it is 100% python

so why does the readme reference asp.net?

Is the readme up to date?

**Comment by ckchessmaster** [comment]
yeah I don't think I ever updated the default readme lol If I revisit this project I'll fix it.
